tests/transforms/test_compile.py

```diff
@@ -33,6 +33,19 @@
 from pennylane.wires import Wires
 
 
+def test_expand_depth_is_deprecated():
+    """Tests that expand_depth is deprecated"""
+    with pytest.warns(
+        qml.PennyLaneDeprecationWarning,
+        match="The expand_depth argument is deprecated",
+    ):
+        ops = (qml.RX(0.1, 0), qml.RX(0.2, 0), qml.Barrier(only_visual=True), qml.X(0), qml.X(0))
+        ms = (qml.expval(qml.X(0)), qml.expval(qml.Y(0)))
+        tape = qml.tape.QuantumScript(ops, ms, shots=50)
+
+        [_], _ = qml.compile(tape, expand_depth=2)
```

Comment:
_Extremely_ nitpicky and optional 😅 , but no need to use a list here.
```suggestion
        _, _ = qml.compile(tape, expand_depth=2)
```